ci/smoketest.sh

```diff
@@ -27,3 +27,22 @@ else
   find $PWD
   exit 1
 fi
+
+# Ensure that client code using appmap.Recording does not fail when APPMAP=false
```

Comment:
This can get moved up before https://github.com/getappmap/appmap-python/pull/327/files#diff-ba2ff002750238ee03b3467fd4400804141c8aa2606bdd4263f0a3636bb120c1R12 to avoid flipping `APPMAP` on and off.

This file is starting to get kind of complicated. Please move this test into a separate function. Thanks!